examples/jsm/nodes/code/ScriptableNode.js

```diff
@@ -0,0 +1,490 @@
+import * as THREE from 'three';
+import * as Nodes from 'three/nodes';
```

Comment:
This actually creates a circular reference... Maybe it's better to do something similar to such function used in NodeLoader.